python/cog/server/runner.py

```diff
@@ -317,6 +344,11 @@ def done(self) -> bool:
         assert self._fut, "call track before checking done"
         return self._fut.done()
 
+    async def wait_async(self) -> None:
```

Comment:
I can guess why you went for this name, but what do you think about `await` to be consistent with the naming convention we've used for async-equivalents (an `a` prefix)?

Reply:
Aha! Fair enough.